DotNet/parser.js

```diff
@@ -178,10 +190,14 @@
 		if (expected.hasOwnProperty('width')) return `new RectangleSize(${expected.width}, ${expected.height})`
 		if (expected.hasOwnProperty('applitools_title')) return `new Region(${expected.applitools_title[0].left}, ${expected.applitools_title[0].top}, ${expected.applitools_title[0].width}, ${expected.applitools_title[0].height})`
 		if (expected.hasOwnProperty('x') && expected.hasOwnProperty('y')) return `new Location(${expected.x}, ${expected.y})`
+		if (expected.hasOwnProperty('name') && expected.hasOwnProperty('value')) return `new PropertyData(\"${expected.name}\", \"${expected.value}\")`
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
		if (expected.hasOwnProperty('name') && expected.hasOwnProperty('value')) {return `new PropertyData(\"${expected.name}\", \"${expected.value}\")`}
```